### ISSUE TITLE
SN Training

### ISSUE THREAD

**Issue by darrenjkt** [CVMI-Lab/ST3D#24]
Hi,

In the original paper for SN, the authors use SN in a fine-tuning process with a detector that has already been pre-trained on the source-to-target task. From looking at the code, the SN implementation in this repo seems to be training from scratch with SN in the data augmentation process - am I right in this understanding? 

Is there a difference in results if you use SN for a train-from-scratch model vs using a pre-trained network and fine-tuning a few more epochs with the SN config?

**Comment by jihanyang** [comment]
Yes, in this repo, we train from scratch with `normalize_object_size` data augmentation.
I tried the source only training and finetuning manner in the very begining of this project. Actually, these two variants obtain similar results in our implementation (Since the code of SN has not been released at that time, the implementation of this augmentation just a reproduction).

**Comment by darrenjkt** [comment]
I see. It's interesting that the results turn out quite similar. Thanks for your response!
